src/FluentCMS.Web.UI/Components/Core/Confirm/Confirm.razor

```diff
@@ -0,0 +1,85 @@
+﻿@inherits BaseComponent
+
+<div class=@this.GetClasses() @attributes=@AdditionalAttributes tabindex="-1">
+
+    @* TODO: It is removed after enabling the JS invoker *@
+    @if (Open)
+    {
+        <div onclick=@Close class="bg-gray-900/50 dark:bg-gray-900/80 fixed inset-0"></div>
+    }
+
+    <ConfirmWrapper>
```

Comment:
@abdolian if `ConfirmWrapper`, `ConfirmContent` and `ConfirmBody` components don't need Props, we can remove those components and add classes by `@this.ClassName("ConfirmWrapper")`... for DOM Elements. to reduce number of components that we should maintain.